### DISCUSSION TITLE
CreateHeaderAndThumbnail.py does not work in Prusa slicer

### DISCUSSION THREAD

**Discussion by H8meCZ** [mriscoc/Ender3V2S1#511] [Q&A]
Hi guys, I am little bit struggling with this script in Prusa slicer 2.5.0. Slicer can not export g-code and it writes error 193. I have properly setup the post-process script link. I have Python and pillow installed in my W10 as well. What can be the root cause of this problem? I am really struggling with this issue because I am not so experienced with python issues and programming. Many thanks!

**Comment by mriscoc** [comment]
Check the troubleshooting section carefully https://github.com/mriscoc/Ender3V2S1/wiki/How-to-generate-a-gcode-preview#troubleshooting
